x11-wm/compiz/compiz-0.8.14.ebuild

```diff
@@ -0,0 +1,119 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils gnome2-utils
```

Comment:
You're not using `eutils` anywhere. You were using `ltprune` but you're asked to remove it.